### ISSUE TITLE
Missing C+23 features

### ISSUE THREAD

**Issue by adolfo-avella** [AnthonyCalandra/modern-cpp-features#137]

**Comment by AnthonyCalandra** [comment]
I've got a lot of work to do ... 😅 This one will take a bit of time.

**Comment by adolfo-avella** [comment]
It is very kind of you to reply anyhow!

**Comment by kcris** [comment]
First of all, thanks for the great great work.

Second, any plans to update for c++ 23? Are you still maintaining this?

Thanks again

**Comment by AnthonyCalandra** [comment]
> First of all, thanks for the great great work.
> 
> Second, any plans to update for c++ 23? Are you still maintaining this?
> 
> Thanks again

Yes I still plan on updating it. I'm definitely a bit behind but I'm also putting in some thought on writing a book that will cover all these in lots more detail than what's in the README. Stay tuned!

**Comment by XRay3D** [comment]
It's time to add C++26)) But the work done is good, I even noticed some things for myself.
